src/NavItem.js

```diff
@@ -1,26 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const renderChildren = (children, href = '') => {
-  if (React.Children.count(children) === 1 && typeof children === 'string') {
-    return <a href={href}>{children}</a>;
-  } else {
-    return React.Children.map(children, (child) =>
```

Comment:
What I mean is we keep the `renderChildren` function, but remove this `map & clone` part.

```js
 if (React.Children.count(children) === 1 && typeof children === 'string') {		
    return <a href={href}>{children}</a>;		
￼ } else {		
￼    return children;
 }

```

WDYT?

Reply:
the problem about this, is that if there's more than one child, ie icon and text it will not be a link anymore and remove functionality

Reply:
that's why I though to just render all passed children

Reply:
Does the styling still work with the `<a>` wrapper?

Reply:
I should think so yes